save-common/src/jvmMain/kotlin/org/cqfn/save/entities/AgentStatus.kt

```diff
@@ -32,16 +34,18 @@ class AgentStatus(
     /**
      * @return this object converted to [AgentStatusDto]
      */
-    fun toDto() = AgentStatusDto(time, state, agent.containerId)
+    fun toDto() = AgentStatusDto(startTime, endTime, state, agent.containerId)
 }
 
 /**
- * @property time time of update
+ * @property startTime staring time of status
+ * @property endTime time of update
  * @property state current state of the agent
  * @property containerId id of the agent's container
  */
 class AgentStatusDto(
-    val time: LocalDateTime,
+    val startTime: LocalDateTime,
+    val endTime: LocalDateTime,
```

Comment:
It doesn't make much sense for me to store both timestamps here. Dto is a thing that orchestrator send to save information on a current heartbeat; start_time should be inserted in DB on state change and not kept on orchestrator